ospfd/ospf_sr.c

```diff
@@ -515,119 +518,104 @@ static int compute_link_nhlfe(struct sr_link *srl)
  *
  * @param srp - Segment Routing Prefix
  *
- * @return -1 if next hop is not found, 0 if nexthop has not changed
- *         and 1 if success
+ * @return -1 if no route is found, 0 if there is no SR route ready
+ *         and 1 if success or update
  */
 static int compute_prefix_nhlfe(struct sr_prefix *srp)
 {
 	struct ospf *top = ospf_lookup_by_vrf_id(VRF_DEFAULT);
-	struct ospf_path *nh = NULL;
+	struct ospf_route *or = NULL;
```

Comment:
I think we should avoid variable names that are keywords.

Reply:
Agree. Change it to `route` in both `ospf_sr.h` and `ospf_sr.c` files